src/structures/Guild.js

```diff
@@ -207,12 +207,42 @@ class Guild extends Base {
     this.joinedTimestamp = data.joined_at ? new Date(data.joined_at).getTime() : this.joinedTimestamp;
 
     /**
-     * The value set for a guild's default message notifications
+     * The value set for the guild's default message notifications
      * @type {DefaultMessageNotifications|number}
      */
     this.defaultMessageNotifications = DefaultMessageNotifications[data.default_message_notifications] ||
       data.default_message_notifications;
 
+    /**
+     * The maximum amount of members the guild can have
+     * @type {number}
+     */
+    this.maximumMembers = data.max_members;
+
+    /**
+     * The maximum amount of presences the guild can have
+     * @type {?number}
+     */
+    this.maximumPresences = data.max_presences;
```

Comment:
Also not present on the Guild Object within the GUILD_CREATE event and `null` implies 5000 [according to Jake](https://canary.discordapp.com/channels/81384788765712384/381887113391505410/485438000394665994)?
```suggestion
    if (typeof data.max_presences !== 'undefined') this.maximumPresences = data.max_presences|| 5000;
```